app/services/usage_thresholds/override_service.rb

```diff
@@ -28,7 +28,6 @@ def call
       result
     rescue ActiveRecord::RecordInvalid => e
       result.record_validation_failure!(record: e.record)
-      result.fail_with_error!
```

Comment:
Notice that this had never worked because it raised a argument error.

Reply:
I guess the intended behavior was use `raise_if_error!`. Not sure why to be honnest